taskvine/src/tools/vine_plot_txn_log

```diff
@@ -545,7 +545,7 @@ if __name__ == "__main__":
     parser.add_argument('-s', action='store_true', help='display tasks sent')
     parser.add_argument('-x', nargs=3, help='change scale for x ticks -x <start> <step_size> <steps>')
     parser.add_argument('-y', nargs=3, help='change scale for y ticks -y <start> <step_size> <steps>')
-    parser.add_argument('-z', nargs=1, help='changes refrence point for 0 Default: manager start (WorkerFirstTask|ManagerFirstTask)')
+    parser.add_argument('-z', nargs=1, help='changes refrence point for 0 Default: manager start (WorkerFirstTask|ManagerFirstTask)', default=["default"])
```

Comment:
Barry, let's change all of these one to long options. (E.g. -x to --xticks, etc.)